scopehal/CMakeLists.txt

```diff
@@ -10,6 +11,11 @@ else()
 	set(LIN_LIBS dl)
 endif()
 
+# adapt for BSD pthreads API
```

Comment:
Also this comment is wrong? This is about `struct stat`, not `pthreads`.

Reply:
that certainly explains my error :D (I was mixing up two things, therefore the "BSD" if)